redisdb/check.py

```diff
@@ -226,6 +226,8 @@ def _check_db(self, instance, custom_tags=None):
         # Save the number of commands.
         self.rate('redis.net.commands', info['total_commands_processed'],
                   tags=tags)
+        self.gauge('redis.net.commands.instantaneous', info['instantaneous_ops_per_sec'],
```

Comment:
This was meant to clarify the fact that it is more or less the same metric, but that commands is taking all ops into account, while commands.instantaneous is just the instantaneous commands per second.
I'm changing that to match redis documentation